test/custom-formatter-test.cc

```diff
@@ -14,8 +14,8 @@ using fmt::BasicPrintfArgFormatter;
 
 // A custom argument formatter that doesn't print `-` for floating-point values
 // rounded to 0.
-class CustomArgFormatter
-  : public fmt::BasicArgFormatter<CustomArgFormatter, char> {
+class CustomArgFormatter :
+    public fmt::BasicArgFormatter<CustomArgFormatter, char> {
```

Comment:
Note that I have changed this one to comply to the one below, which I left untouched from master:

```c++
class CustomPrintfArgFormatter :
    public BasicPrintfArgFormatter<CustomPrintfArgFormatter, char> {
```

I'll correct both then